### PR TITLE
open source blog added

### LINKED ISSUE #1439: [Add] open source blog link
### Site URL

https://sulagnaduttaro.hashnode.dev/open-source-a-beginner-guide-2022

### Description of the above provided site

Learn about open source. A complete guide to beginner's open source journey...

### Category

Open Source

### Sub Category

blogs

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by S-ishita**
## Fixes Issue
closes #1439 

## Screenshots
Screenshot from 2023-07-30 22-55-08

### CODE REVIEW

**Review by CBID2 — APPROVED** [review]
Great job @S-ishita! 😄 LGTM! 😄

**Review by k-deepak04 — DISMISSED** [review]

**Comment by k-deepak04** [comment]
@S-ishita please resolve the conflicts

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm

File: database/open_source/blogs.json [hunk 1]
```diff
@@ -33,5 +33,12 @@
     "url": "https://www.youtube.com/playlist?list=PLnCV5fTAsUgpaLMY_Z8FqPxZ89v-_pcfY",
     "category": "open source",
     "subcategory": "open-source-blogs"
+  },
+  {
+    "name": "Beginner's Guide to Open Source",
+    "description": " A complete guide to beginner's open source journey...",
```
Comment (CBID2) [inline database/open_source/blogs.json:39]:
```suggestion
    "description": " A complete guide to beginner's open source journey.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]

**Review by CBID2 — APPROVED** [review]
Just made a tweak to your description @S-ishita. It looks great now

**Review by rupali-codes — APPROVED** [review]

**Pull request merged**